src/github.com/getlantern/i18n/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
```

Comment:
The i18n code has already been reviewed, this just pulls it into flashlight-build.